scripts/tests/bootstrapper/docker-compose-forest.yml

```diff
@@ -76,15 +76,16 @@ services:
 
         # Make sure to use the Forest bootstrapper as the only bootstrap peer
         cat > config.toml <<EOF
+        [client]
+        data_dir = "/data/forest"
         [network]
         bootstrap_peers = ["/dns/forest-bootstrapper/tcp/$FOREST_P2P_PORT/p2p/$${PEER_ID}"]
         EOF
 
         forest --chain ${CHAIN} --encrypt-keystore false --no-gc \
           --config config.toml \
           --rpc-address 0.0.0.0:${FOREST_RPC_PORT} \
-          --import-snapshot $(ls /data/*.car.zst | tail -n 1) \
-          --import-mode=move
```

Comment:
It was previously copy & delete (which takes 13s in https://github.com/ChainSafe/forest/actions/runs/10783688527/job/29906201147#step:5:121)